Jenkinsfile.nightly

```diff
@@ -0,0 +1,74 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+pipeline {
+    agent {
+        label 'image-test && !master'
+    }
+    tools {
+        maven 'kie-maven-3.6.0'
+        jdk 'kie-jdk1.8'
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 720, unit: 'MINUTES')
+    }
+    stages {
+        stage('Initialize') {
+            steps {
+                sh 'printenv'
+
+            }
+        }
+        stage('Build Nightly projects') {
+            steps {
+                script {
+                    def SETTINGS_XML_ID = "5d9884a1-178a-4d67-a3ac-9735d2df2cef"
+
+                    // This is the map project, variable to store the version from this project
+                    def repositoryListPath = "./script/repository-list.txt"
+                    def projectVariableMap = ['kiegroup_droolsjbpm-build-bootstrap': "kieVersion", 'jboss-integration_izpack': "izpackVersion", 'jboss-integration_installer-commons': "installerCommonsVersion"]
+                    def nightlyStages = load("$WORKSPACE/nightly.stages")
+
+                    configFileProvider([configFile(fileId: "49737697-ebd6-4396-9c22-11f7714808eb", variable: 'PRODUCTION_PROJECT_LIST')]) {
+                        println "Reading file ${repositoryListPath} and merging it with ${PRODUCTION_PROJECT_LIST} jenkins file"
+                        def communityProjectListFile = readFile repositoryListPath
+                        def projectCollection = communityProjectListFile.readLines()
+                        def productionProjectListFile = readFile "${env.PRODUCTION_PROJECT_LIST}"
+                        projectCollection.addAll(productionProjectListFile.readLines())
+                        projectCollection.removeAll { 'droolsjbpm-tools'.equals(it.toLowerCase()) }
+                        println "File ${repositoryListPath} and ${env.PRODUCTION_PROJECT_LIST} jenkins file merged in ${projectCollection}"
+                        nightlyStages(SETTINGS_XML_ID, projectCollection, "rhba/nightly", projectVariableMap)
+                    }
+
+                    def PME_BUILD_VARIABLES = env.PME_BUILD_VARIABLES.split(";").collect{ it.split("=")}.inject([:]) {map, item -> map << [(item.length == 2 ? item[0] : null): (item.length == 2 ? item[1] : null)]}
+
+                    build job: env.RHBA_PROPERTIES_GENERATOR_PATH, parameters: [
+                        [$class: 'BooleanParameterValue', name: 'IS_PROD', value: false],
+                        [$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: PME_BUILD_VARIABLES['productVersion']],
+                        [$class: 'StringParameterValue', name: 'TIME_STAMP', value: "${PME_BUILD_VARIABLES['datetimeSuffix']}"],
+                        [$class: 'StringParameterValue', name: 'ERRAI_VERSION', value: PME_BUILD_VARIABLES['erraiVersion']],
```

Comment:
again Errai version manipulation should be automatical, I guess

Reply:
file removed since it was for testing purposes

Reply:
these are the parameters for the "generate properties files" job. This value comes from the build itself and it's defined on the build-config.yaml file

Reply:
I think properties file doesn't need this anymore

Reply:
removing it from here and from the nightly template. Do I remove it from prod template as well @mareknovotny?